e2e/tests/accessSegregation_test.js

```diff
@@ -2,51 +2,62 @@ const config = require('../config.js');
 
 let caseId;
 
-Feature('Cases visible only to respective local authority and admin');
+Feature('Access segregation');
 
 Before(async (I, caseViewPage, submitApplicationEventPage) => {
-  I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
-  caseId = await I.grabTextFrom('.heading-h1');
-  await I.enterMandatoryFields();
-  caseViewPage.goToNewActions(config.applicationActions.submitCase);
-  submitApplicationEventPage.giveConsent();
-  I.continueAndSubmit();
-  I.wait(2); // in seconds; time needed for access grant calls to complete
-  I.signOut();
+  if (!caseId) {
+    await I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+    await I.enterMandatoryFields();
+    caseViewPage.goToNewActions(config.applicationActions.submitCase);
+    submitApplicationEventPage.giveConsent();
+    I.continueAndSubmit();
+
+    // eslint-disable-next-line require-atomic-updates
+    caseId = await I.grabTextFrom('.heading-h1');
+    console.log(`Case ${caseId} has been submitted`);
```

Comment:
Do we need the console loging?

Reply:
I thought it will give some additional information in case someone wants to grab it and investigate state of the case. I used it few times but it can sometimes be deducted from execution steps (if enabled - not enabled on Jenkins).

It is not ideal when many workers are utilised because you do not know what tests is using created case. I do not feel strong, can remove if you want.

Reply:
I am happy with those.